cmake/OpenCVDetectOpenCL.cmake

```diff
@@ -79,6 +79,19 @@ if(OPENCL_FOUND)
     endif()
   endif()
 
+  if(WITH_CLBLAST)
+    find_path(CLBLAST_INCLUDE_DIR
+              NAMES clblast_c.h
+              HINTS ENV CLBLAST_INSTALL_DIR
```

Comment:
If we want to always use "dynamic loading" then it makes sense to place this header into `3rdparty/include`

Reply:
Yes, it may work but CLBlast version is bumped occassionally along with some tuned results for different devices. If we put a fixed header in `3rdparty/include`, I guess it won't block the new library unless this file has significant changes right?

Reply:
There is just no reason to use DIFFERENT versions of
- header clblast_c.h
- autogenerated files with function from another version of clblast_c.h